tests/common/dualtor/mux_simulator_control.py

```diff
@@ -107,114 +115,145 @@ def _post(server_url, data):
         return False
     return True
 
-def set_drop(mux_server_url, physical_port, directions):
-    """
-    A fixture to set drop for a certain direction on a port
-    Args:
-        mux_server_url: a str, the address of mux server
-        physical_port: physical port on switch, an integer starting from 1
-        directions: a list, may contain "upper_tor", "lower_tor", "nic"
-    Returns:
-        None.
-    """
-    server_url = _url(mux_server_url, physical_port, DROP)
-    data = {"out_ports": directions}
-    pytest_assert(_post(server_url, data), "Failed to set drop on {}".format(directions))
+@pytest.fixture
+def set_drop(url):
```

Comment:
Can both the set_drop and set_output be combined to a single fixture?

Reply:
Same reason as above. I'd like to split them into two function in order to be more friendly to caller.